### DISCUSSION TITLE
[expo-av] Can I Access Raw Audio Data During Recording?

### DISCUSSION THREAD

**Discussion by haryung-lee** [expo/expo#31391] [Archived: Questions]
I am currently using the `expo-av` library and I am wondering if there is a way to access the actual audio data during recording. Specifically, I am looking for a feature similar to what is provided by the [react-native-audio-record library](https://github.com/goodatlas/react-native-audio-record), where you can use a handler like this:

```typescript
AudioRecord.on('data', data => {
  // base64-encoded audio data chunks
});
```

I would like to know if there is an existing way to achieve this with `expo-av`, or if such a feature might be included in a future update.

**Comment by vitordesouza** [comment]
This is becoming more and more important for live transcription and audio processing with the expansion of AI. Anybody from the team can say something why was this archived?

**Comment by Nesh108** [comment]
Throwing in my vote, this is sorely needed today. There is currently no major option for doing this in a managed expo, it will get lots of traction for sure.
